distributed/worker.py

```diff
@@ -1265,14 +1266,14 @@ async def heartbeat(self) -> None:
 
     @fail_hard
     async def handle_scheduler(self, comm: Comm) -> None:
-        await self.handle_stream(comm)
-        logger.info(
-            "Connection to scheduler broken. Closing without reporting. ID: %s Address %s Status: %s",
-            self.id,
-            self.address,
-            self.status,
-        )
-        await self.close(reason="worker-handle-scheduler-connection-broken")
+        try:
+            await self.handle_stream(comm)
+        except CommClosedError:
+            logger.info(
```

Comment:
1. Should be error, shouldn't it?
2. Why not reraise?

Reply:
I was under the impression that this is part of "normal" shutdown. After revisiting the code, it looks like this should not happen and would rather be a concurrency issue that should be properly dealt with.

Reply:
Superseded by https://github.com/dask/distributed/pull/7162#discussion_r1011638692